src/kubernetes_api/core.clj

```diff
@@ -12,6 +12,35 @@
             [martian.httpkit :as martian-httpkit]
             martian.swagger))
 
+(def default-apis
+  "Default API Groups used by Kubernetes.
+   We don't specify versions as we intend to be as future-proof as possible.
+   https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#api-groups"
+  [:admissionregistration.k8s.io
+   :apiextensions.k8s.io
+   :apiregistration.k8s.io
+   :apps
+   :authentication.k8s.io
+   :authorization.k8s.io
+   :autoscaling
+   :batch
+   :certificates.k8s.io
+   :coordination.k8s.io
+   :core
+   :discovery.k8s.io
+   :events.k8s.io
+   :flowcontrol.apiserver.k8s.io
+   :internal.apiserver.k8s.io
+   :networking.k8s.io
+   :node.k8s.io
+   :policy
+   :rbac.authorization.k8s.io
+   :resource.k8s.io
+   :scheduling.k8s.io
+   :storage.k8s.io])
```

Comment:
Explicit set all default apigroups from kubernetes makes us safe against bad APIServices configured in the cluster.